src/test/php/fix-php-compatibility.php

```diff
@@ -123,6 +123,69 @@
             "#[\\ReturnTypeWillChange]\npublic function hasChildren",
         ),
     ),
+    __DIR__ . '/../../../vendor/phpunit/php-code-coverage/src/Report/Html/Renderer/File.php' => PHP_VERSION >= 7 ? array(
```

Comment:
Brackets make it more readable.

```suggestion
    __DIR__ . '/../../../vendor/phpunit/php-code-coverage/src/Report/Html/Renderer/File.php' => (PHP_VERSION >= 7) ? array(
```